hydrus/conf.py

```diff
@@ -37,9 +38,11 @@
     # try to load the path specified in environment variable
     apidoc_env = os.environ['APIDOC_REL_PATH']
     apidoc_path = cwd_path / Path(apidoc_env)
+    FOUND_DOC = True
 except KeyError:
     # if it is not set, use the example doc
-    apidoc_path = cwd_path / 'examples' / 'drones' / 'doc.py'
+    FOUND_DOC = False
+    apidoc_path = cwd_path / 'hydrus' / 'samples' / 'hydra_doc_sample.py'
 
 try:
     APIDOC_OBJ = SourceFileLoader(
```

Comment:
You should make sure the APIDOC specified in the env variable is loaded properly here and then proceed accordingly. 
Rest everything looks fine.

Reply:
I have ensured all the three supported formats can be loaded via environment variables.